dev/inventory/defaults.yaml

```diff
@@ -111,12 +111,13 @@ volumes_defaults:
 node_defaults:
   vcpu: 2
   memory: 4096
-  network_interfaces:
-    - mode: macvtap
-      target: enp42s0
-      mtu: 1500
-      dhcp4: true
-      dhcp6: false
+  network_interfaces: []
+  # network_interfaces:
```

Comment:
Why commented code?

Reply:
Because everyone has a different network interface. I will follow up with a way to customize to the local dev machine, but as it is right now (hardcoded) it is not flexible.